config/crd/bases/skupper_router_access_crd.yaml

```
@@ -10,38 +10,86 @@ spec:
                         type: string
                       port:
+                        description: The port for the router to bind. Must not conflict with another role.
                         type: integer
                     required:
                       - name
                 generateTlsCredentials:
+                  description: |-
+                    When set, Skupper generates the TLS credentials to be
+                    stored in the Secret specified by `tlsCredentials`. See
+                    also `issuer`.
                   type: boolean
                 issuer:
                   type: string
+                  description: |-
+                    The name of the Kubernetes Secret containing the signing CA
+                    used to generate TLS certificates for the RouterAccess when
+                    `generateTlsCredentials` is set.
                 accessType:
+                  description: |-
+                    Configures the access type for the router endpoints.
+                    Available access types and the default selection is
+                    configured on the Skupper controller for Kubernetes.
+
+                    The options available by default are:
+                      `local`: No external ingress. Implies a Kubernetes Service with type CluterIP.
+                      `route`: Exposed via an OpenShift Route.
+                      `loadbalancer`: Exposed via a Kubernetes Service with type LoadBalancer.
                   type: string
                 tlsCredentials:
+                  description: |-
+                    The name of a bundle of TLS certificates used for mutual TLS
+                    router-to-router communication.  The bundle contains the
+                    server certificate and key and the trusted client certificate
+                    (usually a CA).
+
+                    On Kubernetes, the value is the name of a Secret in the
+                    current namespace.
+
+                    On Docker, Podman, and Linux, the value is the name of a
+                    directory under `input/certs/` in the current namespace.
                   type: string
                 bindHost:
+                  description: |-
+                    The hostname or IP address of the network interface to bind
+                    to.  By default, Skupper binds all the interfaces on the host.
                   type: string
                 subjectAlternativeNames:
                   type: array
+                  description: |-
+                    The hostnames and IPs secured by the router TLS certificate.
                   items:
                     type: string
                 settings:
+                  description: |-
+                    Advanced. A map containing additional settings. Each map
+                    entry has a string name and a string value.
+
+                    **Note:** In  general, we recommend not changing settings
```

```suggestion
                    **Note:** In  general, we recommend not changing `settings`
```
maybe this formatting would prevent users/ai from misinterpreting this sentence?